first.js

```diff
@@ -5,6 +5,11 @@ var pwrlvl = function () {
     if (result > 9000) {
         result += " IT'S OVER 9000!!!!!";
     }
+
+    if (result < 2000) {
+        result += "HEUGHHHHH";
```

Comment:
@nickzuber I think 5 `H`s get the point across well enough, is there rationale for a 6th? If not, I would stick with 5 since this already looks good to go.

:shipit: